pkg/packagekit/updates.jsx

```diff
@@ -1323,7 +1323,24 @@ class OsUpdates extends React.Component {
                     icon: "fa fa-exclamation-circle"
                 }
             });
-            return this.state.errorMessages.map(m => <pre key={m}>{m}</pre>);
+
+            return (
+                <>
+                    <EmptyStatePanel title={ STATE_HEADINGS[this.state.state] }
+                                     icon={ ExclamationCircleIcon }
+                                     paragraph={
+                                         <TextContent>
+                                             <Text component={TextVariants.p}>
+                                                 {this.state.errorMessages.map(m => <span key={m}>{m}</span>)}
+                                             </Text>
+                                             <Text component={TextVariants.p}>
+                                                 {_("Try resolving the issue and then refreshing the page")}
```

Comment:
Not sure, not a native speaker myself, but: "Please reload the page after resolving the issue."?

Reply:
It's definitely better than what I had. thanks :)